### DISCUSSION TITLE
Is there a way to disable the Java version warning message?

### DISCUSSION THREAD

**Discussion by cmirzayi** [jagrosh/MusicBot#931] [Questions] [answered]
I'm running the bot on a raspberry pi 2. I installed the correct version of Java, but for some reason I still get a "It appears that you may not be using a supported Java version. Please use 64-bit java." message whenever I start the bot. The bot itself works 100% fine after I hit ok on the message.

Annoyingly this message means I can't setup a script to have the bot run when I startup the pi since I have to hit the ok button on the warning message before the bot will actually start. Is there a way to disable this message box?

**Comment by Sanduhr32** [comment, marked as answer]
Start your bot via the CLI with `-Dnogui=true`. Only the GUI has this warning afaik, CLI mode may print the warning but does not pause until acknowledging it.

**Reply by cmirzayi** [reply]
Oh perfect! Thank you so much!

**Reply by Sanduhr32** [reply]
Optionally you can follow this WIP guide for creating a systemd service, allowing you to autostart jmusicbot.
> https://github.com/Sanduhr32/MusicBot/blob/patch-1/Advanced-Linux-Autostart-Guide.md

**Reply by cmirzayi** [reply]
Oh this is perfect. Thank you!
